src/loc/pt-pt.ts

```diff
@@ -312,6 +316,7 @@ define([], () => {
   "WebSearchLinkLabel": "Pesquisa na Web",
   "Yes": "Sim",
   "SelectedLabel": "Selecionado",
-  "SelectIcon": "Selecionar ícone"
+  "SelectIcon": "Selecionar ícone",
```

Comment:
Should this be moved to the top so that it is listed next to the other strings used by this control?